app/lib/pre_assembly/batch.rb

```diff
@@ -15,7 +15,7 @@ class Batch
                   :objects_had_errors
 
     delegate :staging_location,
-             :content_md_creation,
+             :processing_configuration,
              :content_structure,
```

Comment:
note: `content_md_creation` == `content_metadata_creation` in `BatchContext` ... so we can rename the delegation and then get rid of that level of indirection in the class we are delegating to